static/app/views/issueDetails/streamline/sidebar/detectorSection.tsx

```diff
@@ -0,0 +1,87 @@
+import styled from '@emotion/styled';
+
+import {LinkButton} from 'sentry/components/button';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Event} from 'sentry/types/event';
+import type {Group} from 'sentry/types/group';
+import type {Project} from 'sentry/types/project';
+import {getConfigForIssueType} from 'sentry/utils/issueTypeConfig';
+import useOrganization from 'sentry/utils/useOrganization';
+import {SidebarSectionTitle} from 'sentry/views/issueDetails/streamline/sidebar/sidebar';
+
+interface DetectorDetails {
+  alertRuleId?: string;
+  description?: string;
+}
+
+export function useDetectorDetails({event}: {event: Event}): DetectorDetails {
```

Comment:
are we going to make this more of a hook? right now it could be a function

Reply:
I was thinking about that as well, and if we do things like fetch alert rule details or other info it could be useful, but yeah as written a function is fine. Should I change it?

Reply:
i guess